spec/views/subscriptions_management/index.html.erb_spec.rb

```diff
@@ -3,6 +3,7 @@
     assign(:subscriber, {})
     allow(view).to receive(:use_govuk_account_layout?).and_return(false)
     allow(view).to receive(:is_single_page_subscription?).and_return(false)
+    allow(view).to receive(:get_heading_content).and_return("A thing to subscribe to")
```

Comment:
Another nice thing about helper filers is that they are super easy to unit test.
We can have a  `spec/helper/account_subscriptions_spec.rb` that  just pass a fake subscriber hash in with both bits we need to hit both code branches and cover that off as a simple unit test.